src/pages/Accounts/components/AccountsActionButton.tsx

```diff
@@ -136,28 +137,34 @@ export const AccountsActionButton = ({
         sx={{
           display: 'flex',
           width: '100%',
-          mr: 0.5,
         }}
       >
         <Button
           onClick={onAddNewAccount}
-          sx={{ gap: 1 }}
+          sx={{
+            mr: '1px !important',
```

Comment:
Could we avoid to use the important here and other places?

Reply:
Refactored in [9633fa8](https://github.com/ava-labs/core-extension/pull/94/commits/9633fa819e6c336b0b5d7b443d86be05c66cd517)